### PR TITLE
Update Samples to 1.0 RC1

### PR DESCRIPTION
**Pull request by trgibeau**

### CODE REVIEW

File: samples/SimpleDataSource/SimpleCustomDataProcessor.cs [hunk 1]
```diff
@@ -17,22 +17,22 @@ namespace SampleCustomDataSource
     //
     // This is a sample Custom Data Processor that processes simple text files.
     //
-    // Custom Data Processors are created in Custom Data Sources and are used to actually process the file(s).
-    // An instance of this Processor is created for each set of files opened whereas only one instance of the Custom Data Source is ever created.
-    // Note that CustomDataProcessorBase does not require any file(s) in its constructor, so another
-    // implementation might only store/process one file per instance.
+    // Custom Data Processors are created in Processing Source and are used to actually process the source(s).
```
Comment (mslukebo) [inline samples/SimpleDataSource/SimpleCustomDataProcessor.cs:20]: nit: plurality doesn't match. either use "created by a Processing Source" or "created in Processing Sources"

File: samples/SqlPlugin/SqlPluginWithProcessingPipeline/SqlProcessingSourceWithSourceParser.cs [hunk 2]
```diff
@@ -56,21 +56,19 @@ public override CustomDataSourceInfo GetAboutInfo()
...
+                                                              processorEnvironment);
         }
 
-        protected override bool IsFileSupportedCore(string path)
+        protected override bool IsDataSourceSupportedCore(IDataSource source)
         {
             // Peek inside the XML and make sure our XML namespace is declared and used
 
-            using (var reader = new StreamReader(path))
+            using (var reader = new StreamReader(source.Uri.LocalPath))
```
Comment (mslukebo) [inline samples/SqlPlugin/SqlPluginWithProcessingPipeline/SqlProcessingSourceWithSourceParser.cs:71]: should we be checking that `source is FileDataSrouce`?

**Review by mslukebo — APPROVED** [review]

File: samples/SimpleDataSource/README.md [hunk 3]
```diff
@@ -15,5 +15,5 @@ See `SampleTextFile.txt` for an example of a file with this format.
 2. [.NET Standard 2.0](https://dotnet.microsoft.com/download/visual-studio-sdks)
 
 # Instructions
-Please refer to [Using the SDK/Creating a Simple Custom Data Source](../../documentaiton/Using-the-SDK/Creating-a-simple-custom-data-source.md) for details 
+Please refer to [Using the SDK/Creating a Simple SDK Plugin](../../documentaiton/Using-the-SDK/Creating-a-simple-sdk-plugin.md) for details
```
Comment (anramanath) [inline samples/SimpleDataSource/README.md:18]: I don't think this link works

Comment (mslukebo) [reply samples/SimpleDataSource/README.md:18]: will update in new PR

**Review by anramanath — APPROVED** [review]

**Pull request merged**